### ISSUE TITLE
Release Twisted 15.5

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8050]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8050|
|Type|task|
|Created|2015-10-19 01:18:15Z|
|Branch|https://github.com/twisted/twisted/tree/release-15.5.0-8050-2|

Python 3.5 support, twistd, a bunch of other fun things.

<details><summary>Searchable metadata</summary>

```
trac-id__8050 8050
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__Twisted_15_5 Twisted 15.5
branch__branches_release_15_5_0_8050_2 branches/release-15.5.0-8050-2
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__None None
time__1445217495999093 1445217495999093
changetime__1448776663772337 1448776663772337
version__None None
owner__hawkowl hawkowl
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

(In [46243]) Branching to release-15.5.0-8050.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

(In [46251]) Branching to release-15.5.0-8050-2.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl removed owner|
|-|-|

Builders spun, please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_6"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Do we have a wiki page with reviewer's checklist for a release branch?

Maybe we can update this wiki page https://twistedmatrix.com/trac/wiki/ReleaseProcess with this checklist:

* check that version is ok and news file were updated
* check that all topfiles were removed
* check that trunk has no new merges which add deprecation warnings or remove deprecation code ... or make reference to version number
* check no regression bugs are open ... hopping that people have reported them as regression and not has defect :(

-------

Does this release has a codename ?

Maybe it can be "Goodbye Python 2.6!" so that people will know that python 2.6 is no longer supported.

Of we can have a generic info like this one

https://github.com/twisted/twisted/blob/release-15.5.0-8050-2/NEWS#L2149

--------

Maybe we should also mention that this version is required to have conch.SSH work with latest OpenSSH


-----------

Changes looks good, with a single exception.

I don't understand the changes for [#8070](https://github.com/twisted/twisted/issues/8070).

They are already in trunk but they are listed in the diff... mabye this is just git being git :( and all is ok.

-----

Please check my comments and release + merge.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_7"></a>|@hawkowl commented|
|-|-|

Hi Adi,

We don't, maybe we should add one.

The codename was because there was no quotes last release :) We should add that to the news file though, good idea.

That'll go out in the announcement, I think, but can't hurt to put it with the 2.6 desupporting note.

The [#8070](https://github.com/twisted/twisted/issues/8070) changes are there because github's diffing is weird -- don't worry about them.

I'll add some top-level descriptions, then release and merge.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_8"></a>|@hawkowl set status to `closed`|
|-|-|

(In [46361]) Merge release-15.5.0-8050-2: Release Twisted 15.5

Author: hawkowl
Reviewer: adiroiban
Fixes: [#8050](https://github.com/twisted/twisted/issues/8050)
